api/scpca_portal/common.py

```diff
@@ -22,3 +22,60 @@
 
 IGNORED_INPUT_VALUES = {"", "N/A", "TBD"}
 STRIPPED_INPUT_VALUES = "< >"
+
+# Global sort order for Metadata TSVs
+# Columns
+METADATA_COLUMN_SORT_ORDER = [
+    # sample metadata
+    "scpca_sample_id",
+    "scpca_library_id",
+    "diagnosis",
+    "subdiagnosis",
+    "disease_timing",
+    "age_at_diagnosis",
+    "sex",
+    "tissue_location",
+    "participant_id",
+    "submitter",
+    "submitter_id",
+    "organism",
+    "*",  # Addtional metadata
+    # Library metadata
+    "seq_unit",
+    "technology",
+    "total_reads",
+    "mapped_reads",
+    "sample_cell_count_estimate",
+    "sample_cell_estimates",  # ONLY FOR MULTIPLEXED
+    "unfiltered_cells",
+    "filtered_cell_count",
+    "processed_cells",
+    "has_cellhash",
+    "includes_anndata",
+    "is_cell_line",
+    "is_multiplexed",
+    "is_xenograft",
+    # Project metadata
+    "scpca_project_id",
```

Comment:
lets move this to the first item (thinking about when we have multiple projects)